README.md

```diff
@@ -9,14 +9,17 @@
 
 MAPL is a foundation layer of the GEOS architecture, whose original purpose is to supplement the Earth System Modeling Framework (ESMF).   MAPL fills in missing capabilities of ESMF, provides higher-level interfaces for common boiler-plate logic, and enforces various componentization conventions across ESMF gridded components within GEOS.
 
-MAPL has 7 primary subdirectories for Fortran source code.
+MAPL has 10 primary subdirectories for Fortran source code.
 1. shared - low level utilities that are used throughout the remainder of MAPL.
 2. profiler - time and memory profiling utility
 3. pfio - high-performance client-server I/O layer
-4. generic (under construction) - new home for MAPL extension of ESMF framework.
 5. base (formerly MAPL_Base) - legacy core of MAPL.   This layer will gradually evaporate under further refactoring.
-6. MAPL_cfio - this is a deprecated lower-level I/O layer that is generally replaced by GMAO_pFIO.    Not all of the strings have been cut yet.  Sometime soon, this directory will be eliminated.
-7. griddedio - layer between ESMF container and pfio library
+6. generic (under construction) - new home for MAPL extension of ESMF framework.
+7. oomph - next gen generic will eventually disappear
+8. gridcomps - Cap, History, and ExtData gridcomps used by all GEOS configurations.
+9. MAPL_cfio - this is a deprecated lower-level I/O layer that is generally replaced by GMAO_pFIO.    Not all of the strings have been cut yet.  Sometime soon, this directory will be eliminated.
```

Comment:
Do you want to call it `GMAO_pFIO`? Not sure what the right "name" is. I think in @bena-nasa 's PR for the metadata it was `MAPL_pFIO`. But maybe just "pFIO"?